scripts/src/templates/list-group-item.js

```diff
@@ -2,8 +2,9 @@ export default (data) => (
 `<a href="${data.url}" class="list-group-item${data.selected ? ' active ' : ''} list-group-item-action d-flex justify-content-between align-items-center">
   ${data.title}
   <span>
-    ${data.selected ? '<span class="badge bg-secondary rounded-pill"><i class="fa fa-times"></i></span>' : ''}
-    <span class="badge bg-secondary rounded-pill">${data.count}</span>
+    ${data.selected ? '<span class="badge bg-danger rounded-pill"><i class="fa fa-times"></i></span>' : ''}
+    ${data.selected ? '<span class="badge bg-info rounded-pill">' : '<span class="badge bg-primary rounded-pill">'}
```

Comment:
I think `bg-info` is meant to be paired with `text-dark` to maintain contrast, but can I actually suggest `bg-light text-dark` for both badges? I think that provides sufficient contrast against the selected blue/primary, without drawing too much attention.

Reply:
Oh, and can we update it in `datasets.html` as well, for the server-rendered version?

Reply:
That's much nicer! switching to proposed colors.